rdkit/Chem/EState/Fingerprinter.py

```diff
@@ -35,15 +35,14 @@ def FingerprintMol(mol):
   esIndices = EStateIndices(mol)
 
   nPatts = len(AtomTypes.esPatterns)
-  counts = numpy.zeros(nPatts, numpy.int)
-  sums = numpy.zeros(nPatts, numpy.float)
-
+  counts = [0] * nPatts
+  sums = [0.0] * nPatts
   for i, (_, pattern) in enumerate(AtomTypes.esPatterns):
     matches = mol.GetSubstructMatches(pattern, uniquify=1)
     counts[i] = len(matches)
     for match in matches:
       sums[i] += esIndices[match[0]]
-  return counts, sums
+  return numpy.array(counts, dtype='int'), numpy.array(sums, dtype='float')
```

Comment:
Same argument about using `numpy.zeros` as above.

Reply:
> Same argument about using `numpy.zeros` as above.
As mentioned above: 
I don't agree with it. I changed it intentionally and got accepted in PR 4815.
In particularly, this is a fix of DeprecationWarning in numpy 1.20
Reference: https://numpy.org/devdocs/release/1.20.0-notes.html

Reply:
OK, I got your idea! I would fix it in the next patch. Thanks you for your review.